stepper-2.0.md

```diff
@@ -66,17 +66,16 @@ Protocol
 
 **Stepper step (relative move)**
 
-Position is specified as a 32-bit signed long.
+Steps to move is specified as a 32-bit signed long.
 
 The speed value is a float composed of a 23-bit significand (mantissa) and a 4-bit exponent
```

Comment:
I thought I might mention the custom float format up front, describe it in detail at the end of the document (with examples) and for each method where it's used just refer users to that detailed description.

Sound good?